prow/cmd/horologium/main.go

```diff
@@ -55,9 +75,9 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
-	kc, err := kube.NewClientInCluster(configAgent.Config().ProwJobNamespace)
+	prowJobClient, err := o.kubernetes.ProwJobClient(configAgent.Config().ProwJobNamespace, o.dryRun)
```

Comment:
I wonder what's the difference between
`prowJobClient.List(metav1.ListOptions{LabelSelector: labels.Everything().String()})·
and
`prowJobClient.List(metav1.ListOptions{})`